src/BackgroundTasks/Persistence.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: otruffer
+ * Date: 05.05.17
+ * Time: 10:14
+ */
+namespace ILIAS\BackgroundTasks;
+
+use ILIAS\BackgroundTasks\Implementation\Bucket\State;
+
+interface Persistence {
```

Comment:
Yup, made the whole testing possible. Repository might have been a better name then :-)